code/modules/station_goals/bluespace_tap_lootpools.dm

```diff
@@ -0,0 +1,287 @@
+		/obj/item/dnainjector/remoteview,
+		/obj/item/dnainjector/insulation,
+		/obj/item/dnainjector/eatmut,
+		/obj/item/dnainjector/noprints,
+		/obj/item/dnainjector/comic,
+		/obj/item/seeds/random/labelled,
+		/obj/item/organ/internal/regenerative_core,
+		/obj/item/reagent_containers/glass/bottle/reagent/omnizine,
+		/obj/item/reagent_containers/glass/bottle/reagent/lazarus_reagent
+	)
+
+/obj/effect/spawner/lootdrop/bluespace_tap/organic_uncommon
+	name = "uncommon organic objects"
+	loot = list(
+		/obj/item/storage/pill_bottle/random_meds/labelled,
+		/obj/item/dnainjector/hulkmut,
+		/obj/item/organ/internal/alien/plasmavessel,
+		/obj/item/organ/internal/alien/acidgland,
+		/obj/item/organ/internal/alien/hivenode,
+		/obj/item/organ/internal/alien/neurotoxin,
+		/obj/item/organ/internal/alien/resinspinner,
+		/obj/item/organ/internal/alien/eggsac,
+		/obj/item/organ/internal/heart/gland/heals,
+		/obj/item/organ/internal/heart/gland/ventcrawling,
+		/obj/item/organ/internal/heart/gland/egg,
+		/obj/item/slimepotion/fireproof,
+		/obj/item/slimepotion/oil_slick
+	)
+
+/obj/effect/spawner/lootdrop/bluespace_tap/organic_rare
+	name = "rare organic objects"
+	loot = list(
+		/obj/item/guardiancreator/biological,
+		/obj/item/organ/internal/vocal_cords/adamantine,
+		/obj/item/food/grown/shell/gatfruit,
+		/obj/item/food/grown/cherry_bomb,
+		/obj/item/bio_chip_implanter/storage,
+		/obj/item/slimepotion/sentience,
+		/obj/item/slimepotion/transference
+	)
+
+/obj/effect/spawner/lootdrop/bluespace_tap/food_common
+	name = "common fancy food"
+	lootcount = 1
+	loot = list(
+		/obj/item/food/wingfangchu,
+		/obj/item/food/hotdog,
+		/obj/item/food/cookie,
+		/obj/item/food/plumphelmetbiscuit,
+		/obj/item/food/appletart,
+		/obj/item/food/soup/meatballsoup,
+		/obj/item/food/soup/mysterysoup,
+		/obj/item/food/soup/stew,
+		/obj/item/food/soup/hotchili,
+		/obj/item/food/burrito,
+		/obj/item/food/fishburger,
+		/obj/item/food/cubancarp,
+		/obj/item/food/fishandchips,
+		/obj/item/food/meatpie,
+		/obj/item/pizzabox/hawaiian,
+		/obj/item/food/fortunecookie,
+		/obj/item/food/sugarcookie,
+		/obj/item/food/pie,
+		/obj/item/food/donut/sprinkles,
+		/obj/item/food/chinese/rice,
+		/obj/item/food/pancake,
+		/obj/item/food/chawanmushi,
+		/obj/item/food/syntikebab,
+		/obj/item/food/salad/citrusdelight,
+		/obj/item/food/donkpocket,
+		/obj/item/food/onionrings,
+		/obj/item/food/tatortot,
+		/obj/item/food/stuffing,
+		/obj/item/food/waffles
+	)
+
+/obj/effect/spawner/lootdrop/bluespace_tap/food_uncommon
+	name = "uncommon fancy food"
+	lootcount = 1
+	loot = list(
+		/obj/item/food/sliceable/cheesecake,
+		/obj/item/food/sliceable/bananacake,
+		/obj/item/food/sliceable/chocolatecake,
+		/obj/item/food/sliceable/turkey,
+		/obj/item/food/sliceable/xenomeatbread,
+		/obj/item/food/amanita_pie,
+		/obj/item/food/xemeatpie,
+		/obj/item/food/sliceable/pumpkinpie,
+		/obj/item/food/donut/chaos,
+		/obj/item/food/syndidonkpocket,
```

Comment:
syndi donks should probably be in the rare pool, that or some better foods in the rare pool.

Reply:
Syndi donks are in uncommon due to them, while being good food, having a severe drawback (addiction).

Reply:
yes bug they're *very* good, so wouldn't putting them in rare make more sense?

Reply:
Syndie donks are now rare